detection-rules/brand_impersonation_ms_planner.yml

```diff
@@ -12,6 +12,7 @@ source: |
               or .href_url.domain.root_domain in $free_file_hosts
               or .href_url.domain.root_domain in $free_subdomain_hosts
               or .href_url.domain.domain in $url_shorteners
+              or .href_url.domain.domain in $social_landing_hosts
```

Comment:
Should this also include ?

`or .href_url.domain.root_domain in $social_landing_hosts`